bundle/tests/scorecard/config.yaml

```diff
@@ -70,7 +70,7 @@ stages:
   - entrypoint:
     - cryostat-scorecard-tests
     - operator-install
-    image: quay.io/cryostat/cryostat-operator-scorecard:2.5.0-20240307153901
+    image: quay.io/miwan/cryostat-operator-scorecard:2.5.0-20240314154252
```

Comment:
Looks like the bundle needs to be regenerated without overriding the image to your `miwan` namespace